osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -91,33 +109,31 @@ protected override void PrepareForUse()
 
             prepareDrawables();
 
-            bodyDrawable.ResetAnimation();
+            // not sure if this should remain going forward.
+            skinnableJudgement.ResetAnimation();
 
             this.FadeInFromZero(FadeInDuration, Easing.OutQuint);
-            JudgementBody.ScaleTo(1);
-            JudgementBody.RotateTo(0);
-            JudgementBody.MoveTo(Vector2.Zero);
 
             switch (Result.Type)
             {
                 case HitResult.None:
                     break;
 
                 case HitResult.Miss:
-                    JudgementBody.ScaleTo(1.6f);
-                    JudgementBody.ScaleTo(1, 100, Easing.In);
-
-                    JudgementBody.MoveToOffset(new Vector2(0, 100), 800, Easing.InQuint);
-                    JudgementBody.RotateTo(40, 800, Easing.InQuint);
-
-                    this.Delay(600).FadeOut(200);
+                    ApplyMissAnimations();
                     break;
 
                 default:
                     ApplyHitAnimations();
                     break;
             }
 
+            if (skinnableJudgement.Drawable is IAnimatableJudgement animatable)
+            {
+                using (BeginAbsoluteSequence(Result.TimeAbsolute))
```

Comment:
This won't have any effect because it doesn't have `recurse = true`.

Reply:
should be fixed in the follow-up PR already

Reply:
Indeed it's removed ':D

Reply:
not removed, just much further out in scope.